src/amo/pages/UserProfileEdit/index.js

```diff
@@ -631,7 +636,11 @@ export class UserProfileEditBase extends React.Component<InternalProps, State> {
                           href={user.fxa_edit_email_url}
                           className="UserProfileEdit-manage-account-link"
                         >
-                          {i18n.gettext('Manage Firefox Accounts…')}
+                          {i18n.gettext(
+                            isMzaBranding()
+                              ? 'Manage Mozilla Accounts…'
+                              : 'Manage Firefox Accounts…',
```

Comment:
That isn't going to work. You should have two `gettext` calls otherwise the strings won't be extracted.

(I am surprised that we don't have a linting rule preventing this)